backfill/2022-01-27-sanitize-search-engines/org_mozilla_fenix.sql

```diff
@@ -0,0 +1,149 @@
+CREATE TEMP FUNCTION sanitize_labeled_counter(input ARRAY<STRUCT<key STRING, value INT64>>) AS (
```

Comment:
I'd appreciate some review on this query before starting into this mobile backfill in earnest. See https://github.com/mozilla/bigquery-backfill/pull/17/files#r808469466 for how this is used and for a query for comparison of the results.

cc @sean-rose